lib/macho/load_commands.rb

```diff
@@ -270,8 +270,8 @@ def initialize(lc, lc_str)
           lc_str_abs = view.offset + lc_str
           lc_end = view.offset + lc.cmdsize - 1
           raw_string = view.raw_data.slice(lc_str_abs..lc_end)
-          @string, null_byte, _ = raw_string.partition("\x00")
-          raise LCStrMalformedError.new(lc) if null_byte.empty?
+          @string, null_byte, = raw_string.partition("\x00")
```

Comment:
I'm not liking this resolution. The trailing comma on the left-hand side is too easy to overlook. How about the following?

```ruby
@string, null_byte, _padding = raw_string.partition("\x00")
```